src/app.ts

```diff
@@ -2,10 +2,15 @@ import type { Express } from 'express';
 import cors from 'cors';
 import express from 'express';
 import expressPlayground from 'graphql-playground-middleware-express';
+import { expressLogger, getChildLogger, getModuleBinding } from './logger';
+
+const logger = getChildLogger({ msgPrefix: 'APP' }, { module: getModuleBinding(__filename) });
```

Comment:
If we keep pino, do you want to export a single function `getLogger({ prefix: 'OPTIONAL_PREFIX', module: __filename} )` instead of the two?

Reply:
refactored. no single function is imported and then called with minimum args